packages/core/src/components/menu/menuItem.tsx

```diff
@@ -63,11 +63,11 @@ export class MenuItem extends AbstractPureComponent<IMenuItemProps> {
     };
 
     public render() {
-        const { children, disabled, label, submenu } = this.props;
-        const hasSubmenu = children != null || submenu != null;
-        const liClasses = classNames({
-            [Classes.MENU_SUBMENU]: hasSubmenu,
-        });
+        const { disabled, label } = this.props;
+        const submenuChildren = this.renderSubmenuChildren();
+        const hasSubmenu = submenuChildren != null;
```

Comment:
nit: only used once, can be inlined

Reply:
used twice, or i would have already inlined it 😄  (other usage is hidden in collapsed block, line 66)